cli/lsp/tsc.rs

```diff
@@ -2888,6 +2888,11 @@ impl CompletionEntry {
           }
         }
       }
+      if resolve_url(&source).is_err() {
+        sort_text += "_0";
+      } else {
+        sort_text += "_1";
+      }
```

Comment:
Can you add a comment explaining what this code is doing? Also, shouldn't it be the opposite or I'm probably reading this wrong or don't understand how the sort text works?

Reply:
Lesser sort text means lesser position means higher up on the list. Non-urls (`resolve_url().is_err()`) gives it a lesser sort text suffix.